nixos/doc/manual/development/building-nixos.chapter.md

```diff
@@ -30,6 +30,37 @@ To check the content of an ISO image, mount it like so:
 # mount -o loop -t iso9660 ./result/iso/cd.iso /mnt/iso
 ```
 
+## Notes on additional drivers or firmware {#sec-building-image-drivers}
+
+If you need additional (non-distributable) drivers or firmware in the
+installer, might want to need to extend these configurations.
+
+For example, to build the GNOME graphical installer ISO, but with support for
```

Comment:
You can't build without having a Nix installation at hand.
Maybe run Nix in Docker? https://nixos.org/manual/nix/stable/installation/installing-docker.html

Reply:
This only describes how to build a NixOS installer, and assumes you can invoke Nix somehow.

How to get to this point should probably be described in other parts of the manual.

We might want to add a section "building your own installer" to `nixos/doc/manual/installation/obtaining.chapter.md`, that'll link to the Nix manual (that hopefully describes how to get Nix?)

WDYT?

Reply:
It seems that this whole section shouldn't be in `manual/development` but `installation/` in the first place.

Reply:
Let's get this in as-is. I'll open a followup to move things around.

Reply:
https://github.com/NixOS/nixpkgs/pull/168919